wiki/Tournaments/oMMFF/1/en.md

```diff
@@ -566,6 +594,30 @@ The osu! Multi Mode French Fiesta is run by various community members by distrib
 
 ### osu!mania Division
 
+#### Grand Finals
+
+**[Download the mappack here! (119 MB)](https://mega.nz/#!F5tH2C7L!b4yxO0FwHBJVjsvHnOEhZ-bM27mn34DdwN9Q161fhSQ)**
+
+- FreeMod
+  - [EvOke - Edison (Cunu) \[Singing Vigourously\]](/beatmapsets/924294#mania/1930571)
+  - [Sta - 99 Glooms (MyZterioN-) \[Blooms\]](/beatmapsets/919176#mania/1922832)
+  - [EBIMAYO - GOODRAGE (ARPIA97) \[ARPIA'S HYPER\]](/beatmapsets/677458#mania/1461547)
+  - [Seiryu - waxing and wanding (DanSparce) \[Black Another\]](/beatmapsets/316629#mania/710061)
+  - [Masayoshi Minoshima - Struggle (NekoPoiLTE) \[GRAVITY\]](/beatmapsets/719448#mania/1519286)
+  - [Function Phantom - Euclid (juankristal, XeoStyle) \[Collab\]](/beatmapsets/658097#mania/1393891)
```

Comment:
I guess it's a collab but I thought the purpose of including the creator was just to give people all the metadata they need to identify the map, I'm not sure you need to write XeoStyle here

Reply:
I always include Collab participants and GD Mappers' too, they need their own light too.

Reply:
just seems besides the point for a mappool listing, the creator is only written here to help identify the map, but "XeoStyle" doesn't appear in any of the fields you can see in-game

doesn't really matter either way